### PR TITLE
V2.0.0 alpha.7

### PR DESCRIPTION
**Pull request by danforbes**

### CODE REVIEW

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (dvdplm) [inline tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md:7]:
```suggestion
If you have made it this far, it means you _should_ have a brand new blockchain with custom
```

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (dvdplm) [inline tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md:10]:
```suggestion
Int his tutorial we will build a custom react component that you can add to your `substrate-front-end-template`
```

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (dvdplm) [inline tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md:25]:
```suggestion
You'll see a list of pre-funded accounts and you can make token transfers between those accounts.
```

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (dvdplm) [inline tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md:49]:
```suggestion
Replace the contents of that file with the following:
```

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (dvdplm) [inline tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md:80]:
```suggestion
    // Convert the file content to a hexadecimal representation.
```

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (dvdplm) [inline tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md:124]:
```suggestion
  // The actual UI elements returned from our component.
```

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (dvdplm) [inline tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md:195]:
```suggestion
front-end automatically recognizes that your file is now claimed, and gives you the option to
```

File: tuts/build-a-dapp/v2.0.0-alpha.7/pallet.md
Comment (dvdplm) [inline tuts/build-a-dapp/v2.0.0-alpha.7/pallet.md:20]:
```suggestion
On-chain governance can be added to your blockchain by including pallets like
```

**Review by dvdplm — COMMENTED** [review]
Made a first pass.

**Comment by danforbes** [comment]
@dvdplm - thank you for your suggestions they look spot on! I have contacted you offline to discuss opening a dedicated PR for changes like this so that we can keep this one focused on the upgrade to alpha.7 only :+1:

**Comment by dvdplm** [comment]
> this one focused on the upgrade to alpha.7 only

Ok, I didn't realize that was the plan here. Feel free to ignore my edits. :)

File: tuts/add-a-pallet/v2.0.0-alpha.7/index.md
Comment (JoshOrndorff) [inline tuts/add-a-pallet/v2.0.0-alpha.7/index.md:33]:
```suggestion
Open the `substrate-node-template` in your favorite code editor. We will be editing two files within the runtime (and more in the outer node later on). The two runtime files are:
```

File: tuts/add-a-pallet/v2.0.0-alpha.7/index.md
Comment (JoshOrndorff) [inline tuts/add-a-pallet/v2.0.0-alpha.7/index.md]: The `IMPORTANT` notice is inconsistent with the code that follows. Are we removing the underscore or not?

I vote yes, remove it.

File: tuts/build-a-dapp/v2.0.0-alpha.7/prepare.md
Comment (JoshOrndorff) [inline tuts/build-a-dapp/v2.0.0-alpha.7/prepare.md:35]: **DO NOT MERGE** until after https://github.com/substrate-developer-hub/substrate-front-end-template/pull/90 is merged and tagged.

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (JoshOrndorff) [inline tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md]: Is this space intentional? Is it one of our linting rules? Look funky to my eyes.

File: tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md
Comment (JoshOrndorff) [inline tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md]: You've removed this opening tag, but left the closing tag below.

Were you trying to remove the entire expandable detail box? Or maybe just make it not collapsible?

File: tuts/start-a-private-network/v2.0.0-alpha.7/alicebob.md
Comment (JoshOrndorff) [inline tuts/start-a-private-network/v2.0.0-alpha.7/alicebob.md]: Good work. @thisisvilla will be happy :)

File: tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md
Comment (JoshOrndorff) [inline tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md]: Nice. Now there's no question whether we keep these tutorials boys up-to-date :)

File: tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md
Comment (JoshOrndorff) [inline tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md]: You're quoting the starting point here, right? When I ran the command I still got Bob's usual address, the `5FHnew...` that you removed.

File: tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md
Comment (JoshOrndorff) [inline tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md]: Same

File: tuts/start-a-private-network/v2.0.0-alpha.7/keygen.md
Comment (JoshOrndorff) [inline tuts/start-a-private-network/v2.0.0-alpha.7/keygen.md]: Any particular reason to provide fewer keys? I'm not necessarily opposed to the change, just curious.

**Review by JoshOrndorff — CHANGES_REQUESTED** [review]
Overall looks good @danforbes I pointed up a few little mistakes, then I'm happy to merge it.

File: tuts/add-a-pallet/v2.0.0-alpha.7/index.md
Comment (danforbes) [reply tuts/add-a-pallet/v2.0.0-alpha.7/index.md]: Oh man thanks for pointing this out...good catch!

File: tuts/build-a-dapp/v2.0.0-alpha.7/prepare.md
Comment (danforbes) [reply tuts/build-a-dapp/v2.0.0-alpha.7/prepare.md:35]: This has been merged :heavy_check_mark:

File: tuts/add-a-pallet/v2.0.0-alpha.7/index.md
Comment (danforbes) [reply tuts/add-a-pallet/v2.0.0-alpha.7/index.md:33]: Per my request to David can we keep general verbiage changes out of this PR so that it can remain focused on the upgrade to alpha.7?

File: tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md
Comment (danforbes) [reply tuts/build-a-dapp/v2.0.0-alpha.7/front-end.md]: Yes, I changed the format of this JS so that the linter in the front-end template wouldn't complain about it.

File: tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md
Comment (danforbes) [reply tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md]: Another good catch! I was just removing the tags altogether and guess I just missed this one somehow :sweat_smile: I figured it was the "right thing" to do because we don't want HTML tags but wasn't sure if it was how we wanted the actual page to be formatted. Still not sure on that one.

File: tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md
Comment (danforbes) [reply tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md]: I switched this to use the provided pre-generated keys as example values used throughout.

File: tuts/start-a-private-network/v2.0.0-alpha.7/keygen.md
Comment (danforbes) [reply tuts/start-a-private-network/v2.0.0-alpha.7/keygen.md]: Because we only use two pairs of keys in this tutorial I think it's a bit confusing to provide three pairs of keys as examples. There's already a lot of different types of keys flying around and I feel like this may make it _less_ clear to the user what the right number of the right types of keys are.

File: tuts/add-a-pallet/v2.0.0-alpha.7/index.md
Comment (danforbes) [reply tuts/add-a-pallet/v2.0.0-alpha.7/index.md]: Okay I updated my PR to remove the `IMPORTANT` comment (my preferred approach) and contacted you offline to discuss the best course of action.

File: tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md
Comment (danforbes) [reply tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md]: Removed the closing tag but leaving this conversation unresolved because I'm not sure if this is actually how we want to display this information.

File: tuts/add-a-pallet/v2.0.0-alpha.7/index.md
Comment (JoshOrndorff) [reply tuts/add-a-pallet/v2.0.0-alpha.7/index.md]: Just to state the counterpoint, prefixing a variable name with an underscore typically indicates that it is _not_ used anywhere.

That being said, this is your call and I see the value of just providing a block to copy-paste.

File: tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md
Comment (JoshOrndorff) [reply tuts/create-your-first-substrate-chain/v2.0.0-alpha.7/setup.md]: If we are sticking to the hard rule that no html should be in the markdown files, that I guess this is the best option.

File: tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md
Comment (JoshOrndorff) [reply tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md]: But the text immediately before this bock of code is pointing out what the file the user just generated is supposed to look like.

> The portion of the file we're interested in is the Aura authorities used for creating blocks,
indicated by **"aura"** field below, and GRANDPA authorities used for finalizing blocks, indicated
by **"grandpa"** field. That section looks like this

File: tuts/add-a-pallet/v2.0.0-alpha.7/index.md
Comment (danforbes) [reply tuts/add-a-pallet/v2.0.0-alpha.7/index.md]: Ah see yeah that's a very good point :+1:

Comment (danforbes) [reply tuts/add-a-pallet/v2.0.0-alpha.7/index.md]: Okay, I took my own advice and just reverted this so we can keep this PR focused on the upgrade. We can address potential clarifications separately.

File: tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md
Comment (danforbes) [reply tuts/start-a-private-network/v2.0.0-alpha.7/customspec.md]: Gotcha gotcha...reverted.

**Review by JoshOrndorff — APPROVED** [review]

**Pull request merged**
